generators/app/templates/src/components/Routes/constants.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import React, { lazy } from 'react';
+import { User } from 'contexts/UserContext/reducer';
+
+import PATHS from './paths';
+
+const Home = lazy(() => import('../../screens/Dashboard'));
+
+const MAIN_PUBLIC_PATH = PATHS.login;
+const MAIN_PRIVATE_PATH = PATHS.home;
+
+/* When adding routes, add them ABOVE the Home route
+ * or it will redirect you to incorrect screens */
+export const ROUTES = [
+  // Leaving this as an example for then the Login screen exists
+  {
+    exact: false,
+    path: PATHS.login,
+    component: () => <span>login</span>,
+    title: 'Routes:loginTitle',
+    description: 'Routes:loginDescription',
+    redirectTo: (user: User | null) => (user ? MAIN_PRIVATE_PATH : undefined)
+  },
```

Comment:
I don't think we need this example if we have a generator, maybe we can add a comment here about the generator instead.